astropy/samp/hub.py

```diff
@@ -1136,12 +1282,17 @@ def _reply_(self, responder_private_key, msg_id, response):
             return
 
         responder_public_id = self._private_keys[responder_private_key][0]
-        counter, hub_public_id, recipient_public_id, recipient_msg_tag = msg_id.split(";;", 3)
+        counter, hub_public_id, recipient_public_id, recipient_msg_tag = msg_id.split(
+            ";;", 3
+        )
```

Comment:
Cases like this are unfortunate, though not sure how we can write it more readably

Reply:
Changed in commit 716d079. Feel free to revert it.